charts/pega/templates/pega-environment-config.yaml

```diff
@@ -43,6 +43,12 @@ data:
   JDBC_TIMEOUT_PROPERTIES_RO: {{ .Values.global.jdbc.readerConnectionTimeoutProperties }}
 {{- else }}
   JDBC_TIMEOUT_PROPERTIES_RO: ""
+{{- end }}
+  # compression flag to decompress the config files of Pega Installation.
+{{- if .Values.configurations.isCompressed }}
+  IS_COMPRESSED: "{{ .Values.configurations.isCompressed }}"
+{{- else }}
+  IS_COMPRESSED: "false"
```

Comment:
Variable name looks generic. There is other configuration (jdbc/hazelcast,etc..) in the file. Name does not reflect what configuration being compressed? Can you think of an alternative?

Reply:
yes, Modified the variable name to 'IS_PEGA_CONFIG_COMPRESSED'.